src/box/iproto.cc

```diff
@@ -2978,6 +2977,16 @@ iproto_send_listen_msg(struct evio_service *binary)
 	struct iproto_cfg_msg cfg_msg;
 	iproto_cfg_msg_create(&cfg_msg, IPROTO_CFG_LISTEN);
 	cfg_msg.binary = binary;
+#ifndef NDEBUG
+	struct errinj *inj = errinj(ERRINJ_IPROTO_CFG_LISTEN_FAIL, ERRINJ_BOOL);
```

Comment:
_FAIL suffix is superfluous - please rename the error injection to ERRINJ_IPROTO_CFG_LISTEN.

Reply:
fixed